CONTRIBUTING.md

```diff
@@ -96,6 +96,72 @@ Closes #123
 
 In this case 123 is the corresponding issue number.
 
+### When End-To-End Tests Fail
+
+Every pull request triggers a full run of testing in the CI system.
+The failures reported by the code style checker (aka the "linter") and the unit tests are usually
+clear and easy to fix (and can be avoided by running `npm test` locally before creating a commit).
+But when an integration, or e2e test, fails, it's sometimes useful to consult the log files
+for the run.
+
+Here's how to see more details for a failed run:
+
+1. The test results can be found by first going to the GitHub page for the PR.
+
+  1.1 For a sample, click on an open PR at https://github.com/rancher-sandbox/rancher-desktop/pulls?q=is%3Aopen+is%3Apr
```

Comment:
`1.1`, etc. don't seem to be handled properly in markdown rendering. Consider just doing `1.` here again (but indented).

Reply:
I've been relying on goland's markdown renderer, so I'll push and view in the repo as well before re-requesting the review